src/Spryker/Zed/AppKernel/Business/AppKernelFacade.php

```diff
@@ -19,11 +21,21 @@
  */
 class AppKernelFacade extends AbstractFacade implements AppKernelFacadeInterface
 {
- /**
-  * {@inheritDoc}
-  *
-  * @api
-  */
+    /**
+     * @api
+     *
+     * @inheritDoc
+     */
+    public function validateConfiguration(GlueRequestTransfer $glueRequestTransfer): GlueRequestValidationTransfer
+    {
+        return $this->getFactory()->createConfigurationValidator()->validateConfiguration($glueRequestTransfer);
+    }
```

Comment:
`GlueRequestTransfer` from the communication layer (Glue) leaked to the Business layer. Is there a possibility to pass only form data from the request? Or is it possible to have this plugin on Glue layer?